Kernel/Arch/aarch64/CPUID.h

```diff
@@ -270,5 +270,6 @@ AK_MAKE_ARBITRARY_SIZED_ENUM(CPUFeature, u256,
     __End = CPUFeature(1u) << 255u);
 
 StringView cpu_feature_to_name(CPUFeature::Type const&);
+StringView cpu_feature_to_description(CPUFeature::Type const&);
```

Comment:
Could you also merge this one ( Kernel: Add Aarch64 CPUFeature enumeration description mapping ) into the one that adds the file CPUID.h?

Reply:
yup!